migration-engine/connectors/sql-migration-connector/src/flavour.rs

```diff
@@ -105,6 +105,9 @@ pub(crate) trait SqlFlavour:
         connector: &SqlMigrationConnector,
     ) -> ConnectorResult<SqlSchema>;
 
+    /// Runs a single SQL script.
+    async fn run_query_script(&self, sql: &str, connection: &Connection) -> ConnectorResult<()>;
```

Comment:
I'm still thinking nearly every day about that connection management problem. Do you think it would make sense to have that method on `connection_wrapper::Connection` instead? (genuinely don't know)

Reply:
We could even replace `raw_cmd` there

Reply:
(no need to do that here, maybe I'll give it a try soon)

Reply:
Can we have flavours of that?